actions/create-pr/README.md

```diff
@@ -36,6 +36,13 @@ jobs:
   create_pr:
     runs-on: ubuntu-latest
     steps:
+      - name: 'Create GitHub App Token'
+        uses: actions/create-github-app-token@v1
+        id: generate_token
+        with:
+          app-id: ${{ vars.PR_AUTOMATION_BOT_PUBLIC_APP_ID }}
+          private-key: ${{ secrets.PR_AUTOMATION_BOT_PUBLIC_PRIVATE_KEY }}
```

Comment:
a typo?  `PR_AUTOMATION_BOT_PUBLIC_PRIVATE_KEY` -> `PR_AUTOMATION_BOT_PRIVATE_KEY`

(also below in `README.md`)

Reply:
As we discussed offline, these names are predefined by IDX